CHANGELOG.md

```diff
@@ -13,7 +13,7 @@ When updating the changelog, remember to be very clear about what behavior has c
 and what APIs have changed, if applicable.
 
 ## [Unreleased]
-
+- Improve support for JSR330 by allowing package protected constructors.
```

Comment:
So how exactly does this change behavior from the user's perspective? Does it currently only support public constructors, but now it will support public and package-private (or protected?) constructors? Please help me to understand.

Reply:
Normally for something like this, we would update tests in the PR to demonstrate (as well as test...) the new behavior. If there are any tests for this case, please update them accordingly to demonstrate and test the new behavior.

Reply:
public, protected and package protected constructors are now Injectable instead of only public.

There were no existing tests for this behavior. Happy to add one.

Also, is HEAD passing?


Failure message for com.linkedin.pegasus.generator.TestDataTemplateGeneratorCmdLineApp > testGeneratorWithCustomResolverDirectory:
    Incorrect @Generated annotation, expected: Generated from /var/folders/3w/tzz49g9n0yj2lpcy1938897w000t07/T/TestDataTemplateGeneratorCmdLineApp1631738171784638724531855579560/testWithResolverDirectory.jar:custom/CustomResolverFoo.pdl expected [true] but found [false]

Gradle suite > Gradle test > com.linkedin.pegasus.generator.TestDataTemplateGeneratorCmdLineApp > testGeneratorWithCustomResolverDirectory FAILED
    java.lang.AssertionError at TestDataTemplateGeneratorCmdLineApp.java:176

Failure message for com.linkedin.pegasus.generator.TestDataTemplateGeneratorCmdLineApp > testGeneratorWithMultipleCustomResolverDirectories:
    Incorrect @Generated annotation, expected: Generated from /var/folders/3w/tzz49g9n0yj2lpcy1938897w000t07/T/TestDataTemplateGeneratorCmdLineApp16317381720085544734219936146515/testWithResolverDirectories.jar:custom1/CustomResolverFoo.pdl expected [true] but found [false]

Gradle suite > Gradle test > com.linkedin.pegasus.generator.TestDataTemplateGeneratorCmdLineApp > testGeneratorWithMultipleCustomResolverDirectories FAILED
    java.lang.AssertionError at TestDataTemplateGeneratorCmdLineApp.java:176

Reply:
Discussed offline. Head is passing.